examples/instance/terraform/plugin.go

```diff
@@ -382,8 +382,8 @@ func (p *plugin) Provision(spec instance.Spec) (*instance.ID, error) {
 	// set the tags.
 	// add a name
 	if spec.Tags != nil {
-		if _, has := spec.Tags["Name"]; !has {
-			spec.Tags["Name"] = string(id)
+		if _, has := spec.Tags["name"]; !has {
+			spec.Tags["name"] = string(id)
```

Comment:
Thanks for your comment @ndegory! I've separated this out, so this shouldn't bother AWS. Please take a look at the updated code!